### ISSUE TITLE
Standalone failing on windows

### ISSUE THREAD

**Issue by fpj** [pravega/pravega#4794]
**Problem description**
When I ran `./gradlew startStandalone` from source (`master`), I get an error due to the long classpath:

```
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'C:\Program Files\Java\jdk1.8.0_251\bin\java.exe'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WindowsProcessLauncher.start(WindowsProcessLauncher.java:22)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:67)
        ... 4 more
Caused by: java.io.IOException: Cannot run program "C:\Program Files\Java\jdk1.8.0_251\bin\java.exe" (in directory "C:\Users\junquf\code\pravega\standalone"): CreateProcess error=206, The filename or extension is too long
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
        ... 7 more
Caused by: java.io.IOException: CreateProcess error=206, The filename or extension is too long
        ... 8 more
``` 

**Problem location**
Build, standalone.

**Suggestions for an improvement**
I'm not sure what the fix is, it needs some investigation.

**Comment by ravisharda** [comment]
This issue has been around for a long time now. It happens because the number of characters passed to Windows `CreateProcess` exceeds the max limit owing to the lengthy classpath entries. 

PR #3196 by @sachin-j-joshi [resolved](https://github.com/pravega/pravega/pull/3196#issuecomment-453903801) this issue as well, but the PR wasn't merged, so we still have this issue. 

Here's a workaround I use, whenever I need to run Pravega Standalone in Windows: 

1. Generate the distribution (skipping tests to save time): `gradlew distribution -x test`.
2. Copy config files from `pravega/config` into `pravega/standalone/build/install/pravega-standalone/conf`.
3. Update any configuration as desired under the conf folder.
4. Start Pravega Standalone mode cluster by running the `pravega/standalone/build/install/pravega-standalone/bin/pravega-standalone.bat` file.

**Comment by derekm** [comment]
This is no longer an issue on Windows.

I tested with both `gradlew` shell script on Git Bash for Windows and `gradlew.bat` on cmd.exe.

**Comment by derekm** [comment]
Likely fixed by #5179
